build/context/admin/cdriver/MongoDBCache.cfc

```diff
@@ -32,15 +32,15 @@
 		)>
 
 	<cffunction name="getClass" returntype="string">
-    	<cfreturn "{class}">
+    	<cfreturn "org.lucee.mongodb.cache.MongoDBCache">
```

Comment:
why change this??

Reply:
It didn't seem to work.  My build passed that through to the .lex file as just {class} and the label on the dropdown for the cache list showed up as {label}.  If that was supposed to auto fill from someplace, I can update it so it's correct.

Reply:
Please bear with me as I am a novice at Java programming and this is my first attempt at updating a Lucee extension.  I do have extensive experience with C,C++,C#,Javascript and other similar languages though.  I just am new at the whole .jar file thing.